CHANGELOG.md

```diff
@@ -1,20 +1,14 @@
-# NEXT RELEASE
+# 10.0.0-beta.7 Release notes
 
 ### Enhancements
 * Added ability to replace last sort descriptor on DescriptorOrdering in addition to append/prepending to it [#3884]
 * Add Decimal128 subtraction and multiplication arithmetic operators.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-core/issues/????), since v?.?.?)
-* None.
- 
-### Breaking changes
-* None.
+* Issues fixed by releases v6.0.24 to v6.0.25
```

Comment:
Like the reference to v6 releases. But can we also add the notes here as well so one don't have to look up the notes  from other places?

Reply:
That can be done.

Reply:
@bmunkholm Better now?